hassio/hassio-main.html

```diff
@@ -44,58 +51,77 @@
       ></hass-loading-screen>
     </template>
     <template is='dom-if' if='[[loaded]]'>
-      <iron-pages
-        selected='[[_routeData.page]]'
-        attr-for-selected='page-name'
-        fallback-selection='not-found'
-        selected-attribute='visible'
-      >
-        <hassio-addon-view
-          page-name='addon'
-          route='[[route]]'
-          hass='[[hass]]'
-        ></hassio-addon-view>
-
-        <hassio-addon-store
-          page-name='store'
-          hass='[[hass]]'
-        ></hassio-addon-store>
-
-        <hassio-supervisor
-          page-name='supervisor'
-          hass='[[hass]]'
-          supervisor-info='[[supervisorInfo]]'
-        ></hassio-supervisor>
-
-        <hassio-dashboard
-          page-name='dashboard'
-          hass='[[hass]]'
-          narrow='[[narrow]]'
-          show-menu='[[showMenu]]'
-          supervisor-info='[[supervisorInfo]]'
-          hass-info='[[hassInfo]]'
-        ></hassio-dashboard>
-
-        <hassio-advanced
-          page-name='advanced'
-          hass='[[hass]]'
-          supervisor-info='[[supervisorInfo]]'
-          host-info='[[hostInfo]]'
-          hass-info='[[hassInfo]]'
-        ></hassio-advanced>
-
-        <hassio-snapshots
-          page-name='snapshot'
-          hass='[[hass]]'
-          installed-addons='[[supervisorInfo.addons]]'
-        ></hassio-snapshots>
-
-        <hass-error-screen
-          page-name='not-found'
-          error='Page not found.'
-          title='Hass.io'
-        ></hass-error-screen>
-      </iron-pages>
+      <app-header-layout has-scrolling-region>
+        <app-header fixed slot='header'>
+          <app-toolbar>            
+            <ha-menu-button narrow='[[narrow]]' show-menu='[[showMenu]]'></ha-menu-button>
+            <paper-icon-button
+              icon='mdi:arrow-left'
+              on-tap='backTapped'
+              hidden='[[computeHideBack(mainTitle)]]'></paper-icon-button>
+            <div main-title>[[mainTitle]]</div>
+          </app-toolbar>
+          <paper-tabs
+            scrollable
+            selected='[[routeData.page]]'
+            attr-for-selected='page-name'
+            on-iron-activate='handlePageSelected'
+          >
+            <paper-tab page-name='dashboard'>Dashboard</paper-tab>
+            <paper-tab page-name='store'>Add-on store</paper-tab>
+            <paper-tab page-name='snapshots'>Snapshots</paper-tab>
+            <paper-tab page-name='advanced'>Advanced</paper-tab>
```

Comment:
So let's talk about "Advanced". I had it hidden previously because, you know, it was only for advanced users. Not supposed to be needed. Now it's wide open which means that everyone will take a peek there and be exposed to things that they might not need/want.

When thinking about user experience, think about the things the user wants to accomplish. How do we get them to do that with as little distraction as possible.

Maybe we realized that some functionalities from "Advanced" are better brought to the dashboard, in that case, let's look at them on a per feature basis and make them visible. Exposing all the advanced things like this means that less things can be exposed in the advanced tab because everyone can see everything.

Reply:
"advanced" has some usefull buttons like "shutdown" or "reboot" the host.

Indeed in a first approach we moved stuff from "advanced" to "dashboard" but reverterted this as "dashboard" should be limited to the most important information, -> HA updates and add-on states.

Also if you are using Hass.io and something is not working as expected you have to check the supervisor logs in most cases as the communication of error to the user is not perfect(/good enough atm).

Also the "advanced" tab should maybe show more information in future like device names of USB devices or sound (user can c&p them in their add-on configs where needed) or system stats

Maybe "advanced" should be renamed to "system", idk

Reply:
I like system

Reply:
I also like system. It consists of version info, logs, and restart/reboot options. Nothing I would think of as "advanced".

Reply:
Note that for reboot and stop, @pvizeli has fixed the home Assistant services to call shutdown and stop of hassio. So are they still necessary?

Reply:
The card for home assistant could be removed, leaving only the supervisor and host on the page. It is convenient having it there, but not necessary now that the other options work.  
![image](https://user-images.githubusercontent.com/2456810/36066477-5c3c1b74-0e67-11e8-8b57-1a702c73475d.png)

Reply:
I will change the advanced page in another PR. Maybe we will also get rid of `hassio-data.html`